### PR TITLE
Feature/add priority task

### PR DESCRIPTION
**Pull request by guerracarlos**

### CODE REVIEW

**Comment by miguelGlobant** [comment]
Add color priority to cell backlog.

File: Kanban/Controllers/KBNProjectDetailViewController.m [hunk 1]
```diff
@@ -584,4 +603,23 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     }
 }
 
++ (UIImage *)imageWithColor:(UIColor *)color andSize:(CGSize)size
```
Comment (ignaciozunino) [inline Kanban/Controllers/KBNProjectDetailViewController.m]: please put this in an utility class

File: Kanban/Controllers/Project/KBNProjectTableViewCell.m [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+#import "KBNProjectTableViewCell.h"
+
+@implementation KBNProjectTableViewCell
+
+- (void)awakeFromNib {
+    // Initialization code
+}
+
+- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
```
Comment (ignaciozunino) [inline Kanban/Controllers/Project/KBNProjectTableViewCell.m]: this is doing nothing?

File: Kanban/Controllers/Task/KBNAddTaskViewController.m [hunk 3]
```diff
@@ -33,6 +45,81 @@ - (void)viewDidLoad {
...
+    
+    [priorityData addObject:PRIORITY_HIGH];
+    [priorityData addObject:PRIORITY_MEDIUM];
+    [priorityData addObject:PRIORITY_LOW];
+    
+    float screenWidth = [UIScreen mainScreen].bounds.size.width;
+    float screenHeight = [UIScreen mainScreen].bounds.size.height;
+    float pickerWidth = screenWidth;
+    float pickerHeight = 162.0;
+    float toolBarHeight = 44;
```
Comment (ignaciozunino) [inline Kanban/Controllers/Task/KBNAddTaskViewController.m]: you should be able to take the height from the toolbar

File: Kanban/Controllers/Task/KBNAddTaskViewController.m [hunk 4]
```diff
@@ -33,6 +45,81 @@ - (void)viewDidLoad {
...
+    priorityData = [[NSMutableArray alloc] init];
+    
+    [priorityData addObject:PRIORITY_HIGH];
+    [priorityData addObject:PRIORITY_MEDIUM];
+    [priorityData addObject:PRIORITY_LOW];
+    
+    float screenWidth = [UIScreen mainScreen].bounds.size.width;
+    float screenHeight = [UIScreen mainScreen].bounds.size.height;
+    float pickerWidth = screenWidth;
+    float pickerHeight = 162.0;
```
Comment (ignaciozunino) [inline Kanban/Controllers/Task/KBNAddTaskViewController.m]: constants should be declared as constants at the beginning of the class

File: Kanban/Controllers/Task/KBNTaskDetailViewController.m [hunk 5]
```diff
@@ -28,6 +41,48 @@ - (void)viewDidLoad {
...
+    
+    [priorityData addObject:PRIORITY_HIGH];
+    [priorityData addObject:PRIORITY_MEDIUM];
+    [priorityData addObject:PRIORITY_LOW];
+    
+    float screenWidth = [UIScreen mainScreen].bounds.size.width;
+    float screenHeight = [UIScreen mainScreen].bounds.size.height;
+    float pickerWidth = screenWidth;
+    float pickerHeight = 162.0;
+    float toolBarHeight = 44;
```
Comment (ignaciozunino) [inline Kanban/Controllers/Task/KBNTaskDetailViewController.m]: same as before

File: Kanban/Controllers/Task/KBNTaskDetailViewController.m [hunk 6]
```diff
@@ -28,6 +41,48 @@ - (void)viewDidLoad {
...
+    priorityData = [[NSMutableArray alloc] init];
+    
+    [priorityData addObject:PRIORITY_HIGH];
+    [priorityData addObject:PRIORITY_MEDIUM];
+    [priorityData addObject:PRIORITY_LOW];
+    
+    float screenWidth = [UIScreen mainScreen].bounds.size.width;
+    float screenHeight = [UIScreen mainScreen].bounds.size.height;
+    float pickerWidth = screenWidth;
+    float pickerHeight = 162.0;
```
Comment (ignaciozunino) [inline Kanban/Controllers/Task/KBNTaskDetailViewController.m]: why is this repeated? it is in both the detail and the TaskViewController

File: Kanban/Parse/KBNTaskParseAPIManager.h [hunk 7]
```diff
@@ -13,7 +13,7 @@
 
 @property KBNParseRequestOperationManager* afManager;
 
-- (void)createTaskWithName:(NSString*)name taskDescription:(NSString*)taskDescription order:(NSNumber*)order projectId:(NSString*)projectId taskListId:(NSString*)taskListId completionBlock:(KBNSuccessDictionaryBlock)onCompletion errorBlock:(KBNErrorBlock)onError;
+- (void)createTaskWithName:(NSString*)name taskDescription:(NSString*)taskDescription order:(NSNumber*)order projectId:(NSString*)projectId taskListId:(NSString*)taskListId priotity:(NSNumber *)priority completionBlock:(KBNSuccessDictionaryBlock)onCompletion errorBlock:(KBNErrorBlock)onError;
```
Comment (ignaciozunino) [inline Kanban/Parse/KBNTaskParseAPIManager.h]: it should be "priority" not "priotity". Same everywhere in the rest of the code.

**Comment by ignaciozunino** [comment]
LG

**Pull request merged**